### ISSUE TITLE
Segfault on Stackage build server when running test suite

### ISSUE THREAD

**Issue by snoyberg** [daanx/isocline#1]
On the Stackage build server:

```
Test suite failure for package isocline-1.0.1
    test-example:  exited with: ExitFailure (-11)
Full log available at /var/stackage/work/unpack-dir/.stack-work/logs/isocline-1.0.1-test.log
```

The log file itself is empty. I tested on my local Linux machine, and everything passed correctly, so I'm not sure what the issue is.

I'm going to disable the isocline test suite for Stackage for now.

**Comment by daanx** [comment]
Thanks @snoyberg -- not sure what causes this; it doesn't show up on macOS either. The "test" is strange though as it runs the "interactive" example and relies on ^D/eof to terminate which may perhaps cause issues in an automated pipeline;

so, disabling the test on stackage makes sense for now until we can repro.

**Comment by snoyberg** [comment]
No worries! And sorry I don't have a repro to provide here. Feel free to close this, I tried a few other things with terminals to trigger this and nothing happened.

**Comment by daanx** [comment]
Thanks so much. I think this is fixed since v1.0.7
